shared/src/business/utilities/generateHTMLTemplateForPDF/generateHTMLTemplateForPDF.ts

```diff
@@ -7,29 +7,19 @@
  * @param {object} deconstructed.options optional content that modifies the template
  * @returns {string} hydrated HTML content in string form
  */
-export const generateHTMLTemplateForPDF = async ({
+export const generateHTMLTemplateForPDF = ({
   applicationContext,
   content,
   options = {},
 }) => {
-  const sassContent = require('../htmlGenerator/index.scss_');
+  const main = require('../htmlGenerator/index.scss_');
   const template = require('../htmlGenerator/index.pug_');
 
   const pug = applicationContext.getPug();
   const sass = applicationContext.getNodeSass();
 
-  const { css } = await new Promise((resolve, reject) => {
-    sass.render({ data: sassContent }, (err, result) => {
-      if (err) {
-        applicationContext.logger.error(
-          'Error compiling SASS to CSS while generating PDF',
-          err,
-        );
-        return reject(err);
-      }
-      return resolve(result);
-    });
-  });
+  const { css } = sass.compileString(main);
+
```

Comment:
Can we preserve error logging here?

Reply:
done!

Reply:
https://media.giphy.com/media/Cgo9XkgzRcTPW/giphy.gif

Reply:
![](https://media.giphy.com/media/Cgo9XkgzRcTPW/giphy.gif)